simtools/tests/test_input_validation.py

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/python3
+
+import pytest
+import logging
+
+import simtools.util.input_validation as validator
+
+logger = logging.getLogger()
+logger.setLevel(logging.DEBUG)
+
+
+def test__validate_data_type_datetime():
+
+    date_validator = validator.SchemaValidator(None, None)
+
+    date_key = 'CREATION_TIME'
+    date_schema1 = {'type': 'datetime'}
+
+    # tests should succeed
+    date_validator._validate_data_type(
+        date_schema1, date_key, '2018-03-01 12:00:00')
+
+    # tests should fail
+    date_data1 = '2018-03-01 12:00'
```

Comment:
What happens if you test something like `2018-13-01 12:00`? I assume it fails, but it should be worth a test because that would be the most common mistake I assume.

Reply:
Done. 

(at this point all exceptions are issued as format error; probably something to improve in the future).